### DISCUSSION TITLE
Nuxtjs 4.x (scheduled) what is the different vs Nuxtjs3?

### DISCUSSION THREAD

**Discussion by dbcoder-bold** [nuxt/nuxt#26217] [Questions] [answered]
Hi Guys,

We are planning to migrate from Nextjs2 to Nuxtjs3 and I can see Nuxt 4.x is scheduled for 2024 Q1. 

Therefore I would like to know what is the major changes happing from Nuxtjs3 > Nuxtjs4.

And should we wait for Nuxt4 before migrating to Nuxtjs3?

Thanks for your reply in advanced :)
/Danyal

**Comment by TheAlexLichter** [comment, marked as answer]
No, don't wait. Nuxt 4 will have smaller breaking changes but **nothing** compared to Nuxt 2 -> Nuxt 3. Possible candidate issues [can be found here](https://github.com/nuxt/nuxt/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A4.x).
